docs/src/content/docs/it/resources/plugins.mdx

```diff
@@ -78,6 +78,13 @@ Estendi il tuo sito con plugin ufficiali supportati dal team Starlight e plugin
 		title="starlight-versions"
 		description="Versiona le tue pagine di documentazione Starlight."
 	/>
```

Comment:
Making a suggestion to get this merged before it goes stale. Took the idea from HiDeoo’s French translation given “warp-drive” is not so easy to translate:

```suggestion
	/>
	<LinkCard
		href="https://inox-tools.fryuni.dev/star-warp"
		title="star-warp"
		description="Accedi ai risultati di ricerca per la tua documentazione alla velocità della luce."
	/>
```